### PR TITLE
Add webhook cert-manager docs

### PR DESCRIPTION
**Pull request by adleong**
Add docs describing how to automatically rotate webhook certificates using cert-manager.

Signed-off-by: Alex Leong <alex@buoyant.io>

### CODE REVIEW

**Review by kleimkuhler — APPROVED** [review]
This looks great.

I verified this works with the change in linkerd/linkerd2#5155 as well.

File: linkerd.io/content/2/tasks/automatically-rotating-webhook-tls-credentials.md [hunk 1]
```diff
@@ -0,0 +1,188 @@
...
+helm install linkerd2 \
+  --set installNamespace=false \
+  --set proxyInjector.externalSecret=true \
+  --set-file proxyInjector.caBundle=ca.crt \
+  --set profileValidator.externalSecret=true \
+  --set-file profileValidator.caBundle=ca.crt \
+  --set tap.externalSecret=true \
+  --set-file tap.caBundle=ca.crt \
+  linkerd/linkerd2 \
+  -n linkerd
```
Comment (alpeb) [inline linkerd.io/content/2/tasks/automatically-rotating-webhook-tls-credentials.md:181]: Note that the trust anchor and identity issuer key pair still need to be provided, or else this command will fail.

File: linkerd.io/content/2/tasks/automatically-rotating-webhook-tls-credentials.md [hunk 2]
```diff
@@ -0,0 +1,188 @@
...
+ones that the Linkerd proxies use to secure pod-to-pod communication and use a
+completely separate trust chain.  For more information on rotating the TLS
+credentials used by the Linkerd proxies, see
+[Automatically Rotating Control Plane TLS Credentials](/2/tasks/use_external_certs/).
+
+By default, when Linkerd is installed
+with the Linkerd CLI or with the Linkerd Helm chart, TLS credentials are
+automatically generated for all of the webhooks.  If these certificates expire
+or need to be regenerated for any reason, performing a
+[Linkerd upgrade](/2/tasks/upgrade/) will regenerate them.
```
Comment (alpeb) [inline linkerd.io/content/2/tasks/automatically-rotating-webhook-tls-credentials.md:19]: Can you clarify that both the  CLI and and Helm upgrade automatically regenerate those certs? There's https://linkerd.io/2/tasks/rotating_webhooks_certificates/ explaining the manual rotation procedure, but I'm wondering if that doc really needs to exist if we point out the upgrade procedure here, and then have this doc replace that one...

Comment (adleong) [reply linkerd.io/content/2/tasks/automatically-rotating-webhook-tls-credentials.md]: good call

**Review by alpeb — APPROVED** [review]

**Pull request merged**